pkg/controller/common/resource_reconciler_test.go

```diff
@@ -150,6 +155,7 @@ func TestResourceReconciler_Start(t *testing.T) {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:              "test",
 			DeletionTimestamp: &deletionTimeStamp,
+			Finalizers:        []string{"finalizer"},
```

Comment:
i think we should just remove this test entirely. It's not useful and doesn't actually test anything with these additions

Reply:
dpne